client/src/config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    BASE_URL: "http://localhost:9000",
```

Comment:
Maybe BACKEND_BASE_URL or API_SERVER_BASE_URL? It would probably be helpful to be able to distinguish between the front end and back end URLs, in case there's ever a reason to configure both.

Reply:
Made change.